documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -51,8 +51,15 @@ npm install dotenv
 ...and invoke it before running the built app:
 
 ```diff
--node build
-+node -r dotenv/config build
+- node build
++ node -r dotenv/config build
+```
+
+If you use Node.js v20.6+, you can use the [`--env-file`](https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs) flag instead:
+
+```diff
+- node build
++ node --env-file=.env --env-file=.env.production build
```

Comment:
same as above really. instead of showing how to pass this on build, it would have to be shown when starting the app :thinking: